tcmu-runner.service

```diff
@@ -4,5 +4,5 @@ Description=LIO Userspace-passthrough daemon
 [Service]
 Type=dbus
 BusName=org.kernel.TCMUService1
+KillMode=process
```

Comment:
Hmmm...  Granted it's been a while since I tested this, but I have a trusted/reliable test suite that creates continuous I/O load explicitly checking for data integrity.  I ran that tool on a backstore via tcmu-runner and repeatedly killed tcmu-runner.  The test tool ran perfectly detecting no errors...  That was certainly before cmd_timeout support was added, though.